seed/challenges/02-data-visualization-certification/sass.json

```diff
@@ -1,19 +1,307 @@
+        "<blockquote>div{<br>&nbsp;&nbsp;@include box-shadow(0px, 0px, 4px, #fff)<br>}</blockquote>",
+        "<h4>Instructions</h4>",
+        "Write your own <code>mixin</code> for <code>border-radius</code> and give it a <code>$radius</code> parameter. ",
+        "Then use <code>border-radius</code> to give <code>#awesome</code> a border radius of 10px."
+      ],
+      "challengeSeed": [
+        "<style>",
+        "  /*Add your mixin below this line.*/",
+        "  ",
+        "  /*Add your mixin above this line.*/",
+        "  #awesome{",
+        "    width: 150px;",
+        "    height: 150px;",
+        "    background-color: green;",
+        "    /*Only change code below this line.*/",
+        "",
+        "    /*Only change code above this line.*/",
+        "  }",
+        "</style>",
+        "",
+        "<div id=\"awesome\"></div>",
+        "  "
+      ],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5690af925b3908042c47b32b",
+      "title": "Using @if and @else.",
+      "description": [
+        "The @if directive is usefull if you want to test for a specifc case - it works just like the <code>if</code> statement in Javascript</code>",
+        "<blockquote>@mixin make-bold($bool){<br>&nbsp;&nbsp;@if $bool == bold { font-weight: bold }<br>}</blockquote>",
+        "And just like in Javascript, we can also add <code>@else if</code> and <code>@else</code> to test for more conditions.",
+        "<blockquote>@mixin text-effect($val){<br>&nbsp;&nbsp;@if $val == danger {color: red}<br>&nbsp;&nbsp;@else if $val == alert {color: yellow}<br>&nbsp;&nbsp;@else if $val == success {color: green}<br>&nbsp;&nbsp;@else {color: black}<br>}</blockquote>",
+        "<h4>Instructions</h4>",
+        "Make a <code>mixin</code> called <code>border-stroke</code> that take an argument <code>$val</code>. The mixin should check for the following conditions using <code>@else if</code> and <code>@else</code>:",
+        "<blockquote>light - 1px solid black<br>medium - 3px solid black<br>heavy - 6px solid black<br>none - no border</blockquote>"
+      ],
+      "challengeSeed": [
+        "<style>",
+        "  /*Only change code below this line.*/",
+        "  ",
+        "  /*Only change code above this line.*/",
+        "  #box{",
+        "    width: 150px;",
+        "    height: 150px;",
+        "    background-color: red;",
+        "    @include border-stroke(medium);",
+        "  }  ",
+        "</style>",
+        "",
+        "<div id=\"box\"></div>"
+      ],
+      "tests": [],
+      "type": "waypoint",
+      "challengeType": 0
+    },
+    {
+      "id": "5695eb91ac2df1d813358ab6",
+      "title": "Continuing with @for",
+      "description": [
+        "The <code>@for</code> directive allows you to add styles in a loop - just like a <code>for</code> loop in JS. ",
```

Comment:
I would change it to "JavaScript" for consistency 😜  Everywhere else on the site has it stylized as this.